src/Compilers/CSharp/Test/Syntax/Resources.resx

```diff
@@ -1,6 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
```

Comment:
Was this delete intentional?

Reply:
i didn't do it myself.  looks like perhaps the new resx editor did it.

Reply:
Let's add this back if there is no specific reason to remove it.

Reply:
done.